content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/10-migrate-vm-to-different-node.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Migrate VM to a Different Node"
+metaTitle: "Migrate VM to a Different Node"
+metaDescription: "Learn how to"
+icon: " "
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Overview
+
+Live migration is a process in which a running virtual machine (VM) moves to another cluster node while the guest workload continues to run. During live migration, the VM remains accessible.
+
+Successful live migrations rely on appropriately configured storage and networking and must be enabled as a feature gate. You enable live migration in the ``feature-gates`` field of the kubevirt configuration file in the Spectro VM Dashboard pack.
```

Comment:
This feature gate is enabled by default in our case. Maybe also link it to the feature gates section?

Reply:
Added live migration is enabled by default (It also says so in Feature Gates section) and included a link in this doc to Feature Gates section.

Reply:
done